java/tutorial/README.rst

```diff
@@ -47,13 +46,10 @@ and entry to the java root package. And we could run the Exercise01 with the com
 
 **Exercise 4:** Use ``Ray.wait`` to ignore stragglers.
 
-**Exercise 5:** Enable multiple heterogeneous return values.
+**Exercise 5:** Use multiple heterogeneous return values.
```

Comment:
How about add the link to the file of Exercise*.java, and of course the link will not worked before merged.